server/routerlicious/packages/routerlicious-base/src/utils/sessionHelper.ts

```diff
@@ -59,10 +61,12 @@ async function updateExistingSession(
     deltaStreamUrl: string,
     document: IDocument,
     existingSession: ISession,
-    documentId,
+    documentId: string,
+    tenantId: string,
     documentsCollection: ICollection<IDocument>,
     sessionStickinessDurationMs: number,
     lumberjackProperties: Record<string, any>,
+    count: number,
```

Comment:
Can we remove this variable everywhere?

Reply:
Created a PR here. Thanks for catching it.
https://github.com/microsoft/FluidFramework/pull/13082